### ISSUE TITLE
Problems getting this plugin to work

### ISSUE THREAD

**Issue by flbaue** [insideapp-oss/sonar-flutter#100]
Thanks for this open source plugin, but unfortunately I am struggling to use it.
I am running sonar as part of a Azure DevOps Pipeline with the following config:

```
- task: SonarQubePrepare@5
        condition: eq(variables.testReport, true)
        inputs:
          SonarQube: Sonarqube-MY-mobile
          scannerMode: CLI
          configMode: manual
          cliprojectKey: MY-mobile
          cliprojectName: MY-mobile
          cliprojectVersion: $(build.buildId)
          extraProperties: |
            sonar.projectBaseDir=$(system.defaultWorkingDirectory)/MY-mobile-app
            sonar.sources=lib
            sonar.tests=test,integration_test
            sonar.sourceEncoding=UTF-8
            sonar.log.level=DEBUG
            sonar.verbose=true

- task: SonarQubeAnalyze@5
        condition: eq(variables.testReport, true)
        displayName: Run SonarQube analysis and publish

- task: SonarQubePublish@5
        condition: eq(variables.testReport, true)
        inputs:
          pollingTimeoutSec: '300'

```

I do get a test coverage report, but I do not get the analyzer to work.

1. If I create an analyzer warning on purpose (based on my analisis_options.yaml) it is detected by sonar:
```
2022-01-27T13:53:34.9891900Z 14:53:34.988 INFO: Sensor Dart sensor [flutter] (done) | time=2816ms
2022-01-27T13:53:34.9893170Z 14:53:34.988 INFO: Sensor dartanalyzer sensor [flutter]
2022-01-27T13:53:34.9904520Z 14:53:34.989 INFO: Chosen analyzer mode: flutter
2022-01-27T13:53:34.9905600Z 14:53:34.990 INFO: Running 'flutter analyze'...
2022-01-27T13:53:39.4467990Z 14:53:39.446 INFO: Found issues: 1
2022-01-27T13:53:39.4472920Z 14:53:39.447 DEBUG: Inside issue forEach, file absolute path: /Users/ABC/azure-agent/_work/1/s/MY-mobile-app/app/lib/app/common_widgets/bottom_overlay.dart
2022-01-27T13:53:39.4473930Z 14:53:39.447 WARN: File not included in SonarQube /Users/ABC/azure-agent/_work/1/s/MY-mobile-app/app/lib/app/common_widgets/bottom_overlay.dart
2022-01-27T13:53:39.4474430Z 14:53:39.447 INFO: Sensor dartanalyzer sensor [flutter] (done) | time=4459ms
```
However, the path of the file is wrong. Somehow an extra `.../app/...` was inserted before `/lib`.
Many lines above, the same file was logged like this:
```
2022-01-27T13:53:29.7567540Z 14:53:29.756 DEBUG: 'lib/app/common_widgets/bottom_overlay.dart' indexed with language 'dart'
```

When I run the analyzer locally the output looks like this:
```
➜  MY-mobile-app git:(test-flutter-sonar) ✗ flutter analyze
Analyzing MY-mobile-app...                                            

   info • Use key in widget constructors • lib/app/common_widgets/bottom_overlay.dart:13:3 • use_key_in_widget_constructors

1 issues found. (ran in 4.2s)
```

2. Sonar only detects analyzer issues if they violate my own config. The apparently much stricter options file which is part of this plugin seems not to be used. 

I am currently running out of ideas what I am doing wrong.

**Comment by AndyGettingAn** [comment]
afaik this plugin is currently not working due to the changes made in flutter analyse/coverage output.
Best hope is to wait for an update or spam this feature request for sonarcloud:
[](https://community.sonarsource.com/t/support-for-dart/13555)
